### PR TITLE
jjb/edgex-go/edgex-go-snap.yaml: downgrade california to edge for now

### PR DESCRIPTION
**Pull request by anonymouse64**
Until we do more testing we want the california automated snap builds to be released to edge.

### CODE REVIEW

**Comment by anonymouse64** [comment]
I'd like to hold off on merging this until @tonyespy can confirm the plan

**Comment by anonymouse64** [comment]
After discussion with @tonyespy we want all CI builds to only ever release to edge, and we (meaning the project generally, but for the time being myself, Tony and possibly others) will manually release builds to other risk levels (i.e. `beta`, `candidate`, `stable`) as needed.

**Comment by JPWKU** [comment]
LGTM but I'll wait for Tony to give approval as well.

**Comment by tonyespy** [comment]
@JPWKU Looks good to me.  Just to clarify, this published step is triggered daily, not on every commit to edgex-go, correct?

**Comment by tonyespy** [comment]
Also just to clarify the PR title, we're not downgrading anything here.  Daily builds should always publish to edge.

If/when we have a viable stable candidate, we'll first **manually** publish to the beta or edge channel, and once test results are satisfactory, we'll again manually publish a new version to stable.

**Review by JPWKU — APPROVED** [review]
LGTM

**Pull request merged**
